wpilibc/athena/src/AnalogOutput.cpp

```diff
@@ -49,8 +36,13 @@ AnalogOutput::AnalogOutput(uint32_t channel) {
   HalPortHandle port = getPort(m_channel);
   int32_t status = 0;
   m_port = initializeAnalogOutputPort(port, &status);
-  wpi_setErrorWithContext(status, getHALErrorMessage(status));
   freePort(port);
```

Comment:
What's the intended lifetime of the HalPortHandle?  Here you're freeing it right after we initialize the analog output port on it?